scripts.js

```diff
@@ -0,0 +1,142 @@
+  image: './images/gorillaz.jpeg',
+  subtitle: 'Gorillaz are an English virtual band formed in 1998 by musician',
+  subtitleTwo: ' Damon Albarn and artist Jamie Hewlett, from London, England.',
+  paragraphOne: 'The band primarily consists of four animated members: 2-D ',
+  paragraphTwo: '(vocals, keyboards), Murdoc Niccals (bass guitar), Noodle ',
+  paragraphThree: '(guitar, keyboards, vocals), and Russel Hobbs (drums). ',
+  paragraphFour: 'Their fictional universe is presented in music videos, ',
+  paragraphFive: 'interviews, comic strips and short cartoons.',
+},
+{
+  title: 'Green Day',
+  image: './images/green-day.jpeg',
+  subtitle: 'Green Day is an American rock band formed in the East Bay of',
+  subtitleTwo: ' California in 1987',
+  paragraphOne: 'The band\'s early releases were with the independent record',
+  paragraphTwo: ' label Lookout! Records. In 1994, their major-label debut ',
+  paragraphThree: 'Dookie, released through Reprise Records, became a ',
+  paragraphFour: 'breakout success and eventually shipped over 10 million ',
+  paragraphFive: 'copies in the U.S.',
+},
+{
+  title: 'Jack Johnson',
+  image: './images/jack-johnson.jpeg',
+  subtitle: 'Jack Hody Johnson (born May 18, 1975) is an American ',
+  subtitleTwo: 'singer-songwriter, filmmaker, and former professional surfer.',
+  paragraphOne: 'Johnson has reached number one on the Billboard 200 chart ',
+  paragraphTwo: 'with his albums Sing-A-Longs and Lullabies for the Film ',
+  paragraphThree: 'Curious George in 2006, Sleep Through the Static in 2008,',
+  paragraphFour: ' To the Sea in 2010 and From Here to Now ',
+  paragraphFive: 'to You in 2013.',
+},
+{
+  title: 'RadioHead',
+  image: './images/radio-head.jpeg',
+  subtitle: 'Radiohead are an English rock band formed in Abingdon,',
+  subtitleTwo: ' Oxfordshire, in 1985.',
+  paragraphOne: ' It became a worldwide hit after the release of their debut ',
+  paragraphTwo: 'album, Pablo Honey (1993). Their popularity and critical ',
+  paragraphThree: 'standing rose with the release of their second album,',
+  paragraphFour: ' The Bends ',
+  paragraphFive: '(1995).',
+},
+{
+  title: 'Red Hot Chilli Peppers',
+  image: './images/redhotchilipeppers.jpeg',
+  subtitle: 'The Red Hot Chili Peppers are an American rock band ',
+  subtitleTwo: 'formed in Los Angeles in 1983.',
+  paragraphOne: 'With over 100 million records sold worldwide, the Red Hot ',
+  paragraphTwo: 'Chili Peppers are one of the best-selling bands of all ',
+  paragraphThree: 'time. They are the most successful band in the history',
+  paragraphFour: ' of alternative ',
+  paragraphFive: 'rock.',
+}];
+
+const cardsContainer = document.getElementById('cards-container');
+
+for (let i = 0; i < dataBands.length; i += 1) {
+  const article = document.createElement('article');
+  article.classList.add('band-container');
+
+  article.innerHTML = ` <div class="grid-1">
+    <img src="${dataBands[i].image}" alt="${dataBands[i].title}">
```

Comment:
- [ ] For more consistency with the design you can add the letter squares behind the images and make the images a little bit smaller

|Your design|Figma design|
|--------|-------|
|![Screenshot_20220519_203601](https://user-images.githubusercontent.com/25227022/169388648-d7b4ad29-81cf-4ea5-b574-2716db138e17.png)|![Screenshot_20220519_203824](https://user-images.githubusercontent.com/25227022/169388704-4aed507a-54e2-4b8e-85d2-871520dcda4f.png)|